src/nurse/views.py

```diff
@@ -18,6 +22,77 @@
 )
 
 
+class SendEmailToDoctorView(APIView):
+    def post(self, request, pk):
+        prescription = get_object_or_404(Prescription, id=pk)
+
+        patient = prescription.patient
+        if not patient:
+            return Response(
+                {"error": "This prescription has no patient"},
+                status=status.HTTP_404_NOT_FOUND,
+            )
```

Comment:
Did we cover this case in the tests?

Reply:
No we have not covered this case

Reply:
I added this test test_send_email_to_doctor_no_patient now we cover this case